### ISSUE TITLE
Switch to using SendGrid Marketing Campaign

### ISSUE THREAD

**Issue by TomOrth** [tcnj-acm/hackathon#29]
Related to #19 .

For some reason, this was really bothering me today. So I did some reading on sendgrid. 

There are two reasons I can think of as to why this happened:

1. Sendgrid limits it to 100 emails/day for the free tier
2. Heroku will stop long-running processes (I forget the time limit but its kinda small)

A tool like Mailchimp which provides an API that we can use to create and update mailing lists would be useful. However, to fully support a hackathon, you would need the tier above free which is $15/month.

Luckily, Sendgrid provides a marketing campaign free tier that should suffice for HackTCNJ. See more [here](https://sendgrid.com/solutions/email-marketing/). Sendgrid has a direct API rather than SMTP which should work for sending these campaigns, found [here](https://github.com/sendgrid/sendgrid-ruby).

TL;DR Look into using sendgrid mail campaigns as oppose to the SMTP setup currently in place.
